performance/runner.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+custom_network_name="my_custom_network"
+PEERS=10
+CONNECT_TO=5
+MSG_COUNT=50
+MSG_INTERVAL=100 # ms
+MSG_SIZE=100 # bytes
```

Comment:
not really sure if we really need to pass these as arguments. and it doesn't appear that those will ever need to be parametrized, because `main.nim` will probably never be used at two places simultaneously with different params. params could change over time but only one `main.nim` instance should ever use it.

ether way, it would be nicer to have it code simplified - use `const`s in the `main.nim`. if, there ever would be reason to have arguments it would be easily extended, and for now less code the better.